### PR TITLE
[TACHYON-1361] add javadoc

### PR DESCRIPTION
**Pull request by TeddyBear1314**
I simply add two lines of javadoc before the mTtlBuckets in FileSystemMaster.java

### CODE REVIEW

**Comment by jsimsa** [comment]
Jenkins, test this please

File: servers/src/main/java/tachyon/master/file/FileSystemMaster.java [hunk 1]
```diff
@@ -133,7 +133,11 @@
    */
   @SuppressFBWarnings("URF_UNREAD_FIELD")
   private Future<?> mTtlCheckerService;
-
+  /**
+   * Creates a new list of {@link TtlBucket}s.
```
Comment (jsimsa) [inline servers/src/main/java/tachyon/master/file/FileSystemMaster.java]: this is not a method ... so the JavaDoc should not have `@return` and it should not describe an action, it should describe the purpose of the field

**Comment by jsimsa** [comment]
@TeddyBear1314 thank you for your PR, I left a comment for you ... also please resolve the merge conflicts

**Comment by jsimsa** [comment]
on a related note, your commits are not linked to your github account, please fix that (see https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/)

**Comment by gpang** [comment]
Thanks for your PR! Please resolve conflicts and link the commits to your github account.

**Comment by gpang** [comment]
ok to test

File: servers/src/main/java/tachyon/master/file/FileSystemMaster.java [hunk 2]
```diff
@@ -140,6 +140,10 @@
   @SuppressFBWarnings("URF_UNREAD_FIELD")
   private Future<?> mLostFilesDetectionService;
 
+  /**
+   * This records innodesfiles with ttl set in the corresponding ttlbucket,for the
```
Comment (gpang) [inline servers/src/main/java/tachyon/master/file/FileSystemMaster.java]: rename "records" -> "maintains" and "innodesfiles" -> "inodes"

also, add a space after the comma ","

**Comment by TeddyBear1314** [comment]
Send another PR with commit to make it more clear
https://github.com/amplab/tachyon/pull/2467

**Pull request closed without merging**